server/controllers/authentication/signUp.ts

```diff
@@ -0,0 +1,92 @@
+import { Request, Response } from 'express';
+import nodeMailer from 'nodemailer';
+import dotenv from 'dotenv';
```

Comment:
You don't need to import `dotenv` here or use we just use it in the connection and in the server.